definitions.py

```diff
@@ -113,4 +114,14 @@ class Messages:
     DATASET_NOT_TIME        = "The dataset is not a time vector"
     F_NOT_H5_FILE_FILEPATH  = "f is not h5py.File or filepath to HDF file"
     DATPATH_DOESNT_EXIST    = "{datasetpath} path does not exist in the file"
-    HAS_TO_BE_PATH          = "{path} has to be a path to dataset"
\ No newline at end of file
+    HAS_TO_BE_PATH          = "{path} has to be a path to dataset"
+
+
+
+class Deprecated:
```

Comment:
I would place `class Deprecated` inside the `class DoricFile` above

Reply:
I did that at first but I changed it because I found `defs.Deprecated.DoricFile.Dataset.IMAGES_STACK` better than `defs.DoricFile.Deprecated.Dataset.IMAGES_STACK`. Also I found having only one `class Deprecated` for all the deprecated definition was better.

But  yeah maybe `defs.DoricFile.Deprecated.Dataset.IMAGES_STACK` is better.

Reply:
Do you think we have to track deprecated definitions other than the ones in the doric file?
I feel like `Deprecated.Dataset` makes more sense than `Deprecated.DoricFile`

Reply:
For the moment we only have this dataset to track but later I don't know.

Yes `Deprecated.Dataset` makes more sense